goveralls.go

```diff
@@ -293,24 +294,28 @@ func process() error {
 	} else if prNumber := os.Getenv("BUILDKITE_PULL_REQUEST"); prNumber != "" {
 		pullRequest = prNumber
 	} else if os.Getenv("GITHUB_EVENT_NAME") == "pull_request" {
-		number, _ := getGithubEvent()["number"].(float64)
+		number := getGithubEvent()["number"].(float64)
 		pullRequest = strconv.Itoa(int(number))
 	}
 
 	if *service == "" && os.Getenv("TRAVIS_JOB_ID") != "" {
 		*service = "travis-ci"
 	}
 
+	if *service == "" && os.Getenv("GITHUB_EVENT_NAME") != "" {
+		*service = "GitHub Actions"
```

Comment:
Per my tests, service ["GitHub Actions"](https://github.com/umputun/remark/blob/b03c6a069509b613287f6141973416089f67b495/.github/workflows/ci-build.yml#L58) [doesn't work](https://github.com/umputun/remark/runs/290006811#step:8:67) and ["github"](https://github.com/umputun/remark/blob/e3d0533a9abe36282f9185397e10220cd7e09cab/.github/workflows/ci-build.yml#L58) [do work](https://github.com/umputun/remark/runs/290015994#step:8:67).

Reply:
Oh, sorry.

Reply:
... That's funny. I experience exactly the opposite...

"[github](https://github.com/jderusse/http-broadcast/pull/13/commits/029ae88dc8e056ac2aa5fd6562c1bb98e3bb7163)" [doesn't work](https://github.com/jderusse/http-broadcast/pull/13/checks?check_run_id=290576501) and "[GitHub Actions](https://github.com/jderusse/http-broadcast/pull/13/commits/2f181de67f4823c2502818e14b64b0ef96f88a4e)" [do work](https://github.com/jderusse/http-broadcast/runs/290609740)

Reply:
OK... I got it... that' because of the `COVERALLS_TOKEN`. 

`github` works if COVERALLS_TOKEN = GITHUB_TOKEN
`foo bar baz` works if COVERALLS_TOKEN = THE TOKEN GAVE BY COVERALLS

@mattn what's the right way to fix it ?...  I suggest to remove this code, and explain how it works in the documentation

Reply:
As we discovered in https://github.com/umputun/remark/issues/462, you should use GitHub token in order to be able to access it from fork PR CI run. I think the current solution is good.